pkg/kubelet/kubelet.go

```diff
@@ -222,6 +222,10 @@ func getContainerEtcHostsPath() string {
 	return linuxEtcHostsPath
 }
 
+func ExpiresInTenMinutes(eventTime time.Time, lastUpdate time.Time, maxDuration time.Duration) bool {
+	return eventTime.Sub(lastUpdate) < 600*time.Second
+}
```

Comment:
I don't really like encoding the expiration time in the name here. Maybe just inline it instead?

```go
klet.backoff.HasExpiredFunc = func(eventTime time.Time, lastUpdate time.Time, _ time.Duration) bool {
￼	return eventTime.Sub(lastUpdate) < 10 * time.Minute
}
```

Reply:
Fixed in 815001e